Dangerfile

```diff
@@ -17,8 +17,9 @@ warn('Please add a detailed summary in the description.') if github.pr_body.leng
 declared_trivial = (github.pr_title + github.pr_body).include?('#trivial') || !has_app_changes
 
 if !git.modified_files.include?('CHANGELOG.md') && !declared_trivial
-  raise("Please include a CHANGELOG entry. \nYou can find it at " \
-    '[CHANGELOG.md](https://github.com/ualbertalib/jupiter/blob/master/CHANGELOG.md).', sticky: false)
+  error_message = "Please include a CHANGELOG entry. \nYou can find it at " \
+                  '[CHANGELOG.md](https://github.com/ualbertalib/pushmi_pullyu/blob/master/CHANGELOG.md).'
+  fail(error_message, sticky: false)
```

Comment:
Minor, but variable assignment isn't really needed? Can just stick the string here and style this code accordingly?

Reply:
I am using variable assignment to keep things tidy and readable. If I use the full string in a single line we go over the 120 character limit:
```
  fail("Please include a CHANGELOG entry. \nYou can find it at [CHANGELOG.md](https://github.com/ualbertalib/pushmi_pullyu/blob/master/CHANGELOG.md)", sticky: false)
```
And separating the string with ```\``` inside the call makes it less readable:

```

  fail("Please include a CHANGELOG entry. \nYou can find it at " \
       '[CHANGELOG.md](https://github.com/ualbertalib/pushmi_pullyu/blob/master/CHANGELOG.md).', sticky: false)
```